src/cli/OutputGTFSCommand.ts

```diff
@@ -31,8 +31,14 @@ export class OutputGTFSCommand implements CLICommand {
       throw new Error(`Output path ${this.baseDir} does not exist.`);
     }
 
+    let range = "3 MONTH";
```

Comment:
```suggestion
    const range = process.env["GTFS_RANGE"] || "3 MONTH";
```

Reply:
My thinking was that I wanted a log message to make it clear what was happening

Reply:
okay, add the console.log but leave the assignment of `range` as a one liner